jwql/website/apps/jwql/monitor_pages/monitor_readnoise_bokeh.py

```diff
@@ -107,13 +110,16 @@ class ReadNoisePlotTab():
     def __init__(self, instrument, aperture):
         self.instrument = instrument
         self.aperture = aperture
+        self.ins_ap = "{}_{}".format(self.instrument.lower(), self.aperture.lower())
 
         self.db = ReadnoiseMonitorData(self.instrument, self.aperture)
 
         self.plot_readnoise_amplifers()
         self.plot_readnoise_difference_image()
         self.plot_readnoise_histogram()
 
+        self.file_path = os.path.join(OUTPUT_DIR, "readnoise_monitor", "data", self.ins_ap)
```

Comment:
This  is great @york-stsci along with line 177, this makes sense for serving the static outputs in the app. I think we can use this as a model for moving forward with the other monitors to avoid the issues we were seeing.